pandas/io/parsers/readers.py

```diff
@@ -600,6 +602,14 @@ def _read(
             raise ValueError(
                 "The 'chunksize' option is not supported with the 'pyarrow' engine"
             )
+    elif (
```

Comment:
Was there a reason to disable this for the C/Python parsers? If not, I'm OK with allowing this for consistency purposes.

Given the bugginess/lack of support for some kwargs in the pyarrow engine(Sorry for not really maintaining it after adding it), a user might want to read with the C/Python engine, first.